plugins/plugin-evm/typescript/__tests__/integration/bridge.test.ts

```
@@ -501,16 +499,12 @@ describe("Bridge Action", () => {
 
-          // Validate parameters are reasonable for cost estimation
           expect(parseFloat(bridgeParams.amount)).toBeGreaterThan(0);
           expect(bridgeParams.fromChain).not.toBe(bridgeParams.toChain);
-
-          console.log("Bridge parameters valid for cost estimation");
-        } catch (error) {
-          console.warn("Bridge cost estimation failed:", error);
         }
-      } else {
-        console.warn("Skipping bridge cost estimation - insufficient balance");
+      } catch (error) {
```

Biome (2.4.7)

[error] This variable error is unused.

(lint/correctness/noUnusedVariables)